.github/workflows/simulation-1.2.yml

```diff
@@ -57,6 +57,7 @@ jobs:
       VIRTUAL_TIME: 1
       INTER_OP: 1
       INTER_OP_BBR: 1
+      EXPERIMENTAL_FEATURES: 1
```

Comment:
We have a number of experimental features in OpenThread. It's not clear if/how all those experimental features map to this generic env var. Can we avoid introducing this `EXPERIMENTAL_FEATURES` option in this PR?

Reply:
see above https://github.com/openthread/openthread/pull/9641#discussion_r1435307367 how it came to `EXPERIMENTAL_FEATURES`. Do you suggest adding it in a separate PR or commit?

Reply:
I would propose creating a separate job, as is done with the time sync feature (which is also experimental). For example:
https://github.com/openthread/openthread/blob/e0294176c151134f7bea784ad1557b531808fef0/.github/workflows/simulation-1.1.yml#L190

Reply:
Ok, I have moved the tests to a separate job.

Reply:
The failing tests need builds for `1.1`, which is why https://github.com/openthread/openthread/compare/70f369a6fcb2a0c6b4e4e0953bc28bed6dd8f152..a91e7c3c4b44f9bae64dca9aa9d148477a37150a was needed. Do you see how we can run those tests without `EXPERIMENTAL_FEATURES`? We may rename `EXPERIMENTAL_FEATURES` to `ADDON_FEATURES` and keep running them in a separat job.

Reply:
Rather than updating `script/test`, can we add a check on `OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE` wherever `OPENTHREAD_CONFIG_CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE` is used?

Reply:
@jwhui ok, updated.